src/en/clouds.md

```diff
@@ -88,6 +88,12 @@ If you want more detail about a particular cloud, use:
 juju show-cloud azure
 ```
 
+To learn of any special features a cloud may support the `--include-config`
```

Comment:
I don't really like this here, as it seems to be making something more complicated than it should be, but if it doesn't go here, I am not sure how anyone would know it exists.